source/run-java-queries.txt

```diff
@@ -0,0 +1,62 @@
+.. _intellij-run-java-queries:
+
+================
+Run Java Queries
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use Cases
+---------
+
+The {+intellij-full+} enables you to run queries written in Java directly 
+in the Database Explorer playgrounds. 
+
+The Run button appears next to your MongoDB queries.
+
+.. image:: /images/run-queries/intellij-run-icon.png
+   :alt: Click the Run button to convert your Java query to mongosh syntax.
+
+Behavior
+--------
+
+When you click the Run button, the plugin automatically:
+
+- Converts your Java query to ``mongosh`` syntax. 
+- Maps it directly to your original query. 
+- Opens a Playground file with the populated query.
+
+Example
+-------
+
+The code example below shows the converted query from the code in the 
```

Comment:
I'm not sure if it'd be redundant, but do you think it could be helpful to manually type out the code from the image into its own code block for this Examples section? I think it might be easier for users to have a direct comparison of the two syntaxes next to one another in the same format, whereas for the image, they'd have to scroll up and down + deal with the different visual weight of a screenshot vs. a rendered code block. Thoughts?